config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-periodics.yaml

```diff
@@ -59,7 +59,7 @@ periodics:
     path_alias: "sigs.k8s.io/cluster-api-provider-gcp"
   - org: kubernetes
     repo: kubernetes
-    base_ref: release-1.20
+    base_ref: release-1.22
```

Comment:
please do not change periodic for now, we need to make sure that works for the PRs first